workflows-scripts/inventory/item-record-operations/item_recordOperations.jmx

```diff
@@ -0,0 +1,3811 @@
+            <boolProp name="TransactionController.parent">false</boolProp>
+            <stringProp name="TestPlan.comments">Sorted by unique name or hrid or barcode</stringProp>
+          </TransactionController>
+          <hashTree>
+            <JSR223PreProcessor guiclass="TestBeanGUI" testclass="JSR223PreProcessor" testname="JSR223 PreProcessor" enabled="true">
+              <stringProp name="scriptLanguage">groovy</stringProp>
+              <stringProp name="parameters"></stringProp>
+              <stringProp name="filename"></stringProp>
+              <stringProp name="cacheKey">true</stringProp>
+              <stringProp name="script">String sampleName = sampler.getName();
+
+sampler.setName(&quot;Item_&quot; + vars.get(&quot;pNum&quot;) + &quot;_&quot; + sampleName);</stringProp>
+            </JSR223PreProcessor>
+            <hashTree/>
+            <HeaderManager guiclass="HeaderPanel" testclass="HeaderManager" testname="HTTP Header manager" enabled="true">
+              <collectionProp name="HeaderManager.headers">
+                <elementProp name="X-Okapi-Tenant" elementType="Header">
+                  <stringProp name="Header.name">X-Okapi-Tenant</stringProp>
+                  <stringProp name="Header.value">${tenant}</stringProp>
+                </elementProp>
+                <elementProp name="X-Okapi-Token" elementType="Header">
+                  <stringProp name="Header.name">X-Okapi-Token</stringProp>
+                  <stringProp name="Header.value">${okapi-token}</stringProp>
+                </elementProp>
+                <elementProp name="Content-Type" elementType="Header">
+                  <stringProp name="Header.name">Content-Type</stringProp>
+                  <stringProp name="Header.value">application/json</stringProp>
+                </elementProp>
+                <elementProp name="Accept" elementType="Header">
+                  <stringProp name="Header.name">Accept</stringProp>
+                  <stringProp name="Header.value">application/json</stringProp>
+                </elementProp>
+              </collectionProp>
+            </HeaderManager>
+            <hashTree/>
+            <HTTPSamplerProxy guiclass="HttpTestSampleGui" testclass="HTTPSamplerProxy" testname="GET /search/instances" enabled="true">
+              <elementProp name="HTTPsampler.Arguments" elementType="Arguments" guiclass="HTTPArgumentsPanel" testclass="Arguments" enabled="true">
+                <collectionProp name="Arguments.arguments">
+                  <elementProp name="query" elementType="HTTPArgument">
+                    <boolProp name="HTTPArgument.always_encode">true</boolProp>
+                    <stringProp name="Argument.name">query</stringProp>
+                    <stringProp name="Argument.value">(keyword all &quot;${hrid}&quot; or isbn=&quot;${hrid}&quot; or item.hrid==&quot;${hrid}&quot; or item.id==&quot;${hrid}&quot;) sortby title</stringProp>
```

Comment:
In this query string, _isbn=&quot;${hrid}&quot;_ , should it be a single or double equal signs?  There is a difference in the back end's logic for searching depending on the sign, = or ==.  Please double check. If it's a single = then we should leave it as single.

Reply:
Apparently this HTTPRequest sampler is copied into other test fragments so if there is a change here it should be changed in other places as well.